flatpak_rubygems_generator.rb

```diff
@@ -0,0 +1,53 @@
+# frozen_string_literal: true
```

Comment:
Is it any different from https://github.com/flatpak/flatpak-builder-tools/blob/master/rubygems/flatpak_rubygems_generator.rb ? If not, just remove it or add as git submodule for convenience.